MacPass/de.lproj/Localizable.strings

```diff
@@ -69,23 +69,23 @@
 "AUTOTYPE_YES" = "Autotype aktivieren";
 
 /* Sucessfully merged external changes */
-"AUTO_MERGE_NOTIFICATION_TEXT" = "Daten wurden erfolgreich automatische synchronisiert";
+"AUTO_MERGE_NOTIFICATION_TEXT" = "Daten wurden erfolgreich automatisch synchronisiert";
 
 /* Cancel */
 "CANCEL" = "Abbrechen";
 
 /* Menu item in the database outline context menu to change the database name */
-"CHANGE_DATABASE_NAME" = "Name der Datenbank ändern";
+"CHANGE_DATABASE_NAME" = "Namen der Datenbank ändern";
 
 /* (No Comment) */
-"CHANGE_FORMAT" = "Als KDBX speichern";
+"CHANGE_FORMAT" = "Als .KDBX speichern";
```

Comment:
I would be against introducing the dot. This is about the file format not the file extension.